vsintegration/tests/FSharp.Editor.Tests/DocumentDiagnosticAnalyzerTests.fs

```diff
@@ -129,16 +129,6 @@ let b =
             expectedMessage = "Incomplete structured construct at or before this point in binding"
         )
 
-    [<Fact>]
-    member public this.Error_Type_WithoutName() =
-        this.VerifyErrorBetweenMarkers(
-            fileContents =
-                """
-type (*start*)=(*end*)
```

Comment:
This code produces two errors now, and I did not find a way to update the test case for it.